HISTORY.rst

```diff
@@ -20,6 +20,12 @@ Internal Changes
 ~~~~~~~~~~~~~~~~
 * `xclim` code quality checks now use the newest `black` (v21.8-beta). Checks launched via `tox` and `pre-commit` now run formatting modifications over Jupyter notebooks found under `docs`.
 
+New features and enhancements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* utils.py::nan_calc_percentiles now uses a custom algorithm instead of `numpy::nanpercentiles` to have more flexibility on the interpolation method. The performances are also improved. *
```

Comment:
```suggestion
* ``utils.py::nan_calc_percentiles`` now uses a custom algorithm instead of ``numpy::nanpercentiles`` to have more flexibility on the interpolation method. The performance is also improved. 
```